[2021-01-23 18:26:42,494][MainThread:4742960576][task_id:django.utils.autoreload][autoreload.py:612][INFO][Watching for file changes with StatReloader]
[2021-01-23 18:27:46,175][Thread-1:123145353228288][task_id:django.request][log.py:224][WARNING][Not Found: /]
[2021-01-23 18:28:09,897][Thread-2:123145370017792][task_id:django.request][log.py:224][WARNING][Not Found: /]
[2021-01-23 18:28:12,487][Thread-1:123145353228288][task_id:django.request][log.py:224][WARNING][Not Found: /test]
[2021-01-23 18:28:12,547][Thread-2:123145370017792][task_id:django.request][log.py:224][WARNING][Not Found: /favicon.ico]
[2021-01-23 18:28:17,701][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (0/20) now.]
[2021-01-23 18:28:17,701][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (1/20) in 1.00 second.]
[2021-01-23 18:28:18,703][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (2/20) in 1.00 second.]
[2021-01-23 18:28:19,707][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (3/20) in 1.00 second.]
[2021-01-23 18:28:20,710][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (4/20) in 1.00 second.]
[2021-01-23 18:28:21,715][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (5/20) in 1.00 second.]
[2021-01-23 18:28:22,718][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (6/20) in 1.00 second.]
[2021-01-23 18:28:23,722][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (7/20) in 1.00 second.]
[2021-01-23 18:28:24,724][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (8/20) in 1.00 second.]
[2021-01-23 18:28:25,728][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (9/20) in 1.00 second.]
[2021-01-23 18:28:26,730][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (10/20) in 1.00 second.]
[2021-01-23 18:28:27,733][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (11/20) in 1.00 second.]
[2021-01-23 18:28:28,737][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (12/20) in 1.00 second.]
[2021-01-23 18:28:29,743][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (13/20) in 1.00 second.]
[2021-01-23 18:28:30,744][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (14/20) in 1.00 second.]
[2021-01-23 18:28:31,750][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (15/20) in 1.00 second.]
[2021-01-23 18:28:32,754][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (16/20) in 1.00 second.]
[2021-01-23 18:28:33,759][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (17/20) in 1.00 second.]
[2021-01-23 18:28:34,763][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (18/20) in 1.00 second.]
[2021-01-23 18:28:35,767][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (19/20) in 1.00 second.]
[2021-01-23 18:28:36,771][Thread-1:123145353228288][task_id:celery.backends.redis][redis.py:132][CRITICAL][
Retry limit exceeded while trying to reconnect to the Celery redis result store backend. The Celery application must be restarted.
]
[2021-01-23 18:28:39,956][MainThread:4742960576][task_id:django.utils.autoreload][autoreload.py:236][INFO][/Users/lihong/Downloads/dispatch/djcelery/core/settings.py changed, reloading.]
[2021-01-23 18:28:40,679][MainThread:4578043328][task_id:django.utils.autoreload][autoreload.py:612][INFO][Watching for file changes with StatReloader]
[2021-01-23 18:29:38,548][Thread-1:123145349152768][task_id:django.request][log.py:224][WARNING][Not Found: /test]
[2021-01-23 18:29:38,701][Thread-1:123145349152768][task_id:django.request][log.py:224][WARNING][Not Found: /favicon.ico]
[2021-01-23 18:29:43,783][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (0/20) now.]
[2021-01-23 18:29:43,783][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (1/20) in 1.00 second.]
[2021-01-23 18:29:44,788][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (2/20) in 1.00 second.]
[2021-01-23 18:29:45,792][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (3/20) in 1.00 second.]
[2021-01-23 18:29:46,797][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (4/20) in 1.00 second.]
[2021-01-23 18:29:47,801][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (5/20) in 1.00 second.]
[2021-01-23 18:29:48,806][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (6/20) in 1.00 second.]
[2021-01-23 18:29:49,810][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (7/20) in 1.00 second.]
[2021-01-23 18:29:50,814][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (8/20) in 1.00 second.]
[2021-01-23 18:29:51,818][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (9/20) in 1.00 second.]
[2021-01-23 18:29:52,823][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (10/20) in 1.00 second.]
[2021-01-23 18:29:53,828][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (11/20) in 1.00 second.]
[2021-01-23 18:29:54,833][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (12/20) in 1.00 second.]
[2021-01-23 18:29:55,837][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (13/20) in 1.00 second.]
[2021-01-23 18:29:56,842][Thread-1:123145349152768][task_id:celery.backends.redis][redis.py:362][ERROR][Connection to Redis lost: Retry (14/20) in 1.00 second.]
[2021-01-23 18:31:43,619][MainThread:4462724544][task_id:django.utils.autoreload][autoreload.py:612][INFO][Watching for file changes with StatReloader]
[2021-01-23 18:31:49,064][Thread-1:123145405530112][task_id:js][views.py:31][ERROR][test eror]
[2021-01-23 18:31:49,064][Thread-1:123145405530112][task_id:tf][views.py:32][ERROR][test eror]
[2021-01-23 18:31:49,064][Thread-1:123145405530112][task_id:tf][views.py:32][ERROR][test eror]
[2021-01-23 18:31:49,064][Thread-1:123145405530112][task_id:logstash][views.py:33][ERROR][test eror]
[2021-01-23 19:05:57,857][MainThread:4589954496][task_id:django.utils.autoreload][autoreload.py:612][INFO][Watching for file changes with StatReloader]
[2021-01-23 19:06:00,416][Thread-1:123145390387200][task_id:django.request][log.py:224][WARNING][Not Found: /]
[2021-01-23 19:06:06,141][Thread-2:123145407176704][task_id:django.request][log.py:224][WARNING][Not Found: /]
[2021-01-23 19:06:06,695][Thread-2:123145407176704][task_id:django.request][log.py:224][WARNING][Not Found: /test]
[2021-01-23 19:06:08,856][Thread-1:123145390387200][task_id:js][views.py:31][ERROR][test eror]
[2021-01-23 19:06:08,856][Thread-1:123145390387200][task_id:tf][views.py:32][ERROR][test eror]
[2021-01-23 19:06:08,856][Thread-1:123145390387200][task_id:tf][views.py:32][ERROR][test eror]
[2021-01-23 19:06:08,856][Thread-1:123145390387200][task_id:logstash][views.py:33][ERROR][test eror]
[2021-01-23 19:06:08,980][Thread-2:123145407176704][task_id:django.request][log.py:224][WARNING][Not Found: /favicon.ico]
[2021-01-23 19:08:21,370][MainThread:4589954496][task_id:django.utils.autoreload][autoreload.py:236][INFO][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py changed, reloading.]
[2021-01-23 19:08:23,001][MainThread:4698539456][task_id:django.utils.autoreload][autoreload.py:612][INFO][Watching for file changes with StatReloader]
[2021-01-23 19:08:27,229][Thread-1:123145457909760][task_id:js][views.py:32][ERROR][test eror-1611400107.229214]
[2021-01-23 19:08:27,229][Thread-1:123145457909760][task_id:tf][views.py:33][ERROR][test eror]
[2021-01-23 19:08:27,229][Thread-1:123145457909760][task_id:tf][views.py:33][ERROR][test eror]
[2021-01-23 19:08:27,230][Thread-1:123145457909760][task_id:logstash][views.py:34][ERROR][test eror]
[2021-01-23 19:08:29,084][Thread-1:123145457909760][task_id:js][views.py:32][ERROR][test eror-1611400109.084568]
[2021-01-23 19:08:29,085][Thread-1:123145457909760][task_id:tf][views.py:33][ERROR][test eror]
[2021-01-23 19:08:29,085][Thread-1:123145457909760][task_id:tf][views.py:33][ERROR][test eror]
[2021-01-23 19:08:29,085][Thread-1:123145457909760][task_id:logstash][views.py:34][ERROR][test eror]
[2021-01-23 19:42:56,711][MainThread:4698539456][task_id:django.utils.autoreload][autoreload.py:236][INFO][/Users/lihong/Downloads/dispatch/djcelery/core/settings.py changed, reloading.]
[2021-01-23 19:42:58,433][INFO][django.utils.autoreload][/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/django/utils/autoreload.py][autoreload.py][run_with_reloader:612][Watching for file changes with StatReloader]
[2021-01-23 19:44:52,526][INFO][django.utils.autoreload][/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/django/utils/autoreload.py][autoreload.py][trigger_reload:236][/Users/lihong/Downloads/dispatch/djcelery/core/settings.py changed, reloading.]
[2021-01-23 19:45:54,610][INFO][django.utils.autoreload][/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/django/utils/autoreload.py][autoreload.py][run_with_reloader:612][Watching for file changes with StatReloader]
[2021-01-23 19:46:01,072][ERROR][js][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:32][test eror-1611402361.072254]
[2021-01-23 19:46:01,072][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 19:46:01,072][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 19:46:01,072][ERROR][logstash][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:34][test eror]
[2021-01-23 20:25:33,072][ERROR][js][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:32][test eror-1611404733.071845]
[2021-01-23 20:25:33,074][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 20:25:33,074][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 20:25:33,074][ERROR][logstash][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:34][test eror]
[2021-01-23 20:25:35,368][ERROR][js][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:32][test eror-1611404735.368054]
[2021-01-23 20:25:35,368][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 20:25:35,368][ERROR][tf][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:33][test eror]
[2021-01-23 20:25:35,368][ERROR][logstash][/Users/lihong/Downloads/dispatch/djcelery/orders/views.py][views.py][get:34][test eror]
